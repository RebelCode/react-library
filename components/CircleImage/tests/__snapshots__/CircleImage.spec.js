import React from "react";
import { render, cleanup } from "@testing-library/react";
import { CircleImage } from "../../index";

afterEach(cleanup);

describe("CircleImage", () => {
  it("should take a snapshot", () => {
    const { asFragment } = render(
      <CircleImage aria-label="CircleImage" />
    );

    expect(asFragment(<CircleImage />)).toMatchSnapshot();
  });

  it("renders correctly", () => {
    const { queryByLabelText } = render(<CircleImage aria-label="CircleImage" />);

    expect(queryByLabelText("CircleImage")).toBeTruthy();
  });

  it("renders text correctly", () => {
    const { queryByLabelText } = render(
      <CircleImage aria-label="CircleImage" />
    );

    expect(queryByLabelText("CircleImage")).toBeTruthy();
  });
});
