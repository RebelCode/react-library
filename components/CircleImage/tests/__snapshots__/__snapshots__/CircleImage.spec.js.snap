// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`CircleImage should take a snapshot 1`] = `
<DocumentFragment>
  <img
    aria-label="CircleImage"
    class="image "
  />
</DocumentFragment>
`;
